web/src/components/vis/CorrelationTile.vue

```diff
@@ -89,13 +90,27 @@ export default {
   },
   methods: {
     searchFilter(item, queryText, itemText) {
-      const match = itemText.toLocaleLowerCase().indexOf(queryText.toLocaleLowerCase()) > -1;
-      if (match) this.searchBarResults.add(itemText);
-      else this.searchBarResults.delete(itemText);
+      const match = itemText.toLocaleLowerCase().indexOf(queryText.toLocaleLowerCase()) > -1
+                    && !this.excludedSearchBarResults.has(itemText);
+      if (match) {
+        this.searchBarResults.add(itemText);
+      } else this.searchBarResults.delete(itemText);
```

Comment:
Probably should wrap this in brackets.  I know you just followed my bad formatting, sorry.